wiki/Tournaments/RESC/2024/en.md

```diff
@@ -0,0 +1,539 @@
+---
+tags:
+  - ResCup
+  - Res Cup
+  - Serenity
+---
+
+# Resurrection Cup 2024
+
+![Resurrection Cup 2024](img/banner.jpg)
+
+**Resurrection Cup 2024** was a 4v4, double-elimination osu! tournament hosted by ::{ flag=HU }:: [Phreel](https://osu.ppy.sh/users/12840110) and ::{ flag=VN }:: [Hoaq](https://osu.ppy.sh/users/7696512). It was the third instalment in the Resurrection Cup series.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2024-05-11/2024-06-01 |
+| Screening phase | 2024-06-01/2024-06-08 |
+| Qualifiers showcase | 2024-06-09 |
+| Qualifiers | 2024-06-14/2024-06-16 |
+| Round of 32 | 2024-06-21/2024-06-23 |
+| Round of 16 | 2024-06-28/2024-06-30 |
+| Quarterfinals | 2024-07-05/2024-07-07 |
+| Semifinals | 2024-07-12/2024-07-14 |
+| Finals | 2024-07-19/2024-07-21 |
+| Grand Finals | 2024-07-26/2024-07-28 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge, 70% of the prizepool, exclusive profile banner |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 20% of the prizepool, exclusive profile banner |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 10% of the prizepool, exclusive profile banner |
+| 4th place | Exclusive profile banner |
```

Comment:
not sure if this is right tbh, i can see this done most recently on the [COE wiki page](https://osu.ppy.sh/wiki/en/Tournaments/COE/COE_2024)
```suggestion
| "4th place" | Exclusive profile banner |
```